website/static/js/contribManager.js

```diff
@@ -59,15 +42,32 @@ var ContributorModel = function(contributor, currentUserCanEdit, pageOwner, isRe
     var self = this;
     $.extend(self, contributor);
 
+    self.permissionList = [
+        {value: 'read', text: 'Read'},
+        {value: 'write', text: 'Read + Write'},
+        {value: 'admin', text: 'Administrator'}
+    ];
+    self.getPermission = function(permission) {
```

Comment:
This function is brittle (it would break if the order of `permissionList` changed). Better to search through the Array for where `permission` equals the value of `value`.